### ISSUE TITLE
isElementInWater in vehicles

### ISSUE THREAD

**Issue by FileEX** [multitheftauto/mtasa-blue#2545]
**Describe the bug**
isElementInWater returns false if ped or player is in vehicle - Returns true only for the vehicle. 
**This bug occurs only on client side**

**To reproduce**
1. Get vehicle
2. Drive into the water 
3. 
```lua
print(isElementInWater(me)) -- false
print(isElementInWater(me.vehicle)) -- true
```

**Expected behaviour**
isElementInWater should returns true also for peds/players which are in vehicle in water

**Screenshots**

**Version**
1.5.9

**Additional context**

**Comment by Fernando-A-Rocha** [comment]
Can be closed, fix is part of r21160

**Comment by lopezloo** [comment]
Fixed in 29f303860e5404aa98a2a69ca5a3b6e80eebd2df
